packages/ui/src/NetworkIndicator.tsx

```diff
@@ -5,43 +5,39 @@ interface NetworkIndicatorProps {
   bitcoinRpcUrl: string;
 }
 
+const getNetworkDetails = (network: string, isMutinynet: boolean) => {
+  const networkDetails: { [key: string]: { color: string; name: string } } = {
+    bitcoin: { color: '#FF9900', name: 'Mainnet' },
+    main: { color: '#FF9900', name: 'Mainnet' },
+    testnet: { color: '#6BED33', name: 'Testnet' },
+    test: { color: '#6BED33', name: 'Testnet' },
+    signet: isMutinynet
+      ? { color: 'red', name: 'Mutinynet' }
+      : { color: 'purple', name: 'Signet' },
+    regtest: { color: '#33C6EC', name: 'Regtest' },
+    default: { color: 'gray', name: 'Unknown' },
+  };
+
+  return networkDetails[network] || networkDetails['default'];
+};
+
+const isMutinynet = (bitcoinRpcUrl: string) => {
+  try {
+    const url = new URL(bitcoinRpcUrl);
+    return url.host === 'mutinynet.com' && url.pathname === '/api';
```

Comment:
This still doesn't work, do you just want to drop the pathname and make it mutinynet if it's from mutinynet.com?

![CleanShot 2023-12-05 at 12 19 13@2x](https://github.com/fedimint/ui/assets/649992/6a01bac0-b7de-488e-9d94-9f40ebe1b6a3)